packages/@o3r/design/schematics/extract-token/index.ts

```diff
@@ -46,7 +46,7 @@ function extractTokenFn(options: ExtractTokenSchematicsSchema): Rule {
 
   return async (tree, context) => {
     try {
-      // eslint-disable-next-line @typescript-eslint/naming-convention
+      // eslint-disable-next-line @typescript-eslint/naming-convention -- convention for class name
```

Comment:
I would put something like `false positive when using dynamic import` cause we deactivate this rule for imports
and we should check if an issue is already opened or if we should open one :)